drivers/ethernet/eth_xlnx_gem.c

```diff
@@ -647,17 +620,25 @@ static enum ethernet_hw_caps eth_xlnx_gem_get_capabilities(
 		caps |= ETHERNET_HW_TX_CHKSUM_OFFLOAD;
 	}
 
-	if (dev_conf->enable_fdx) {
-		caps |= ETHERNET_DUPLEX_SET;
-	}
-
 	if (dev_conf->copy_all_frames) {
 		caps |= ETHERNET_PROMISC_MODE;
 	}
 
```

Comment:
similar with ``ETHERNET_PROMISC_MODE``, this should tell, if ETHERNET_CONFIG_TYPE_PROMISC_MODE is supported in set_config

Reply:
Resolved via the other PR that introduced the set_config function.